index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Water Con</title>
```

Comment:
Water Con? Is this the name of your app? What does water con mean? :man_shrugging:

Reply:
@matthewdking `Water-you-talking-about` :see_no_evil:

Reply:
ohhhhh yeah let's change this

Reply:
i'll be guessing it's short for Consumption

Reply:
@matthewdking 
> Water Con? Is this the name of your app? What does water con mean?

Fixed :+1: :smile: